search/utils.py

```diff
@@ -282,12 +331,18 @@ def create_mapping():
     """
 
     # Create the index if it doesn't exist.
-    conn = get_conn()
-    if not conn.indices.exists(INDEX_NAME):
-        conn.indices.create(INDEX_NAME)
+    get_conn()
+
+
+def _create_mapping():
```

Comment:
A small nit, but it may look cleaner to pass in conn as a parameter. As it is the function assumes `_CONN` will already exist when `_create_mapping` is called, and if it were a parameter the assumption would be made explicit.